contracts/Migrator.sol

```diff
@@ -7,6 +7,8 @@ import { IERC20Like } from "./interfaces/Interfaces.sol";
 
 contract Migrator {
 
+    uint256 public constant SCALER = 10;
```

Comment:
let's make this immutable and set in the constructor please

Reply:
let's rename to TOKEN_SPLIT_SCALAR, scalar alone is too generic (also note the spelling with an A)